### PR TITLE
add database license icons

### PR DESCRIPTION
**Pull request by pgwillia**
Parse the mangled string stored in Solr to get the icon attributes. 
Display the icon on the results and show catalog pages.

screenshot from 2019-01-25 12-14-21
screenshot from 2019-01-25 12-17-44
screenshot from 2019-01-25 12-15-39

#1113

### CODE REVIEW

**Comment by pgwillia** [comment]
@seanluyk What do you think about the dual use of the UAL shield? https://github.com/ualbertalib/discovery/issues/1113#issuecomment-456509823

File: app/views/catalog/_show_header_default.html.erb [hunk 1]
```diff
@@ -17,4 +17,12 @@
       </div>
     <% end %>
   </div>
+  <p>
+    <strong>Conditions of Use:</strong>
```
Comment (weiweishi) [inline app/views/catalog/_show_header_default.html.erb]: Do we need to worry about the i18n in blacklight?

**Comment by pgwillia** [comment]
@seanluyk Yes!  The descriptions are used as the alt value and images link out to their description.  Like this
```
<a href="https://www.library.ualberta.ca/about-us/policies/restricted-resource-info">
          <img alt="licensed for  U of A" src="https://libapps-ca.s3.amazonaws.com/sites/165/icons/6665/ualib-logo.png" height="46">
</a>
```

File: app/views/catalog/_show_header_default.html.erb [hunk 2]
```diff
@@ -17,4 +17,18 @@
...
     <% end %>
   </div>
+  <div class='container'>
+    <div class='row row-no-gutters'>
+      <h2 class='col-md-3 h3 align-middle'><%= t('conditions_of_use') %>:</h2>
+      <ul class='col-md-9 align-middle list-inline'>
+      <% libguides_icons(document).each do |icon| -%>
+        <li class='list-inline-item'>
+        <%= link_to icon[:url] do %>
+          <%= image_tag(libguide_icon_image(icon), alt: icon[:description], height: '30') %>
```
Comment (murny) [inline app/views/catalog/_show_header_default.html.erb]: One small thing that could improve UX is we could add a `title` attr to the image tags? Be the same content as the `alt` attr

Why? This adds a nice tooltip when someone hovers over the icons giving some additional information of what the icons actually mean (aka hovering over the UofA shield icon will show a tooltip with `licensed for U of A`)
```suggestion
          <%= image_tag(libguide_icon_image(icon), alt: icon[:description], title: icon[:description], height: '30') %>
```

File: app/views/catalog/_show_header_default.html.erb [hunk 3]
```diff
@@ -17,4 +17,18 @@
       </div>
     <% end %>
   </div>
+  <div class='container'>
```
Comment (murny) [inline app/views/catalog/_show_header_default.html.erb]: Shouldn't need to redefine a container div here? This whole partial gets rendered into a container already

Comment (pgwillia) [reply app/views/catalog/_show_header_default.html.erb]: When trying to get the description and the images to line up the Bootstrap documentation seemed to suggest that it was required.  I'm not sure it is either.  I'll check and remove if it's not.

Comment (murny) [reply app/views/catalog/_show_header_default.html.erb]: Yeah you need a container for rows/cols but we already have a container (every single thing in blacklight gets rendered inside a container that comes from the application layout: https://github.com/ualbertalib/discovery/blob/master/app/views/layouts/application.html.erb#L47)

Hope that makes sense can explain in more detail offline

**Pull request merged**